tests/peft_integration/test_peft_integration.py

```diff
@@ -265,17 +265,31 @@ def test_peft_add_multi_adapter(self):
                 _ = model.generate(input_ids=dummy_input)
 
                 model.set_adapter("default")
-                self.assertTrue(model.active_adapter() == "default")
+                self.assertTrue(model.active_adapter() == ["default"])
+                self.assertTrue(model.active_adapter(return_multi_adapters=False) == "default")
 
                 model.set_adapter("adapter-2")
-                self.assertTrue(model.active_adapter() == "adapter-2")
+                self.assertTrue(model.active_adapter() == ["adapter-2"])
+                self.assertTrue(model.active_adapter(return_multi_adapters=False) == "adapter-2")
 
                 # Logits comparison
                 self.assertFalse(
                     torch.allclose(logits_adapter_1.logits, logits_adapter_2.logits, atol=1e-6, rtol=1e-6)
                 )
                 self.assertFalse(torch.allclose(logits_original_model, logits_adapter_2.logits, atol=1e-6, rtol=1e-6))
 
+                model.set_adapter(["adapter-2", "default"])
+                self.assertTrue(model.active_adapter() == ["adapter-2", "default"])
```

Comment:
Nice addition to the test. Can we also check `model.active_adapter()` and `model.active_adapter(return_multi_adapters=False)` here?